saleor/payment/utils.py

```diff
@@ -277,9 +277,15 @@ def _create_refund_manual_amount(
 
     Takes into account previous refunds and current refund mutation parameters.
     """
-    previous_manual_amount_to_refund = order.fulfillments.filter(
-        lines__order_line__variant_id__isnull=True, shipping_refund_amount__isnull=True
-    ).aggregate(discount=Sum("total_refund_amount"))["discount"]
+    previous_manual_amount_to_refund = (
+        Fulfillment.objects.prefetch_related("lines__order_line")
+        .filter(
+            order_id=order.pk,
+            lines__order_line__variant_id__isnull=True,
+            shipping_refund_amount__isnull=True,
```

Comment:
I don't think that we should rely on the shipping_refund_amount as a value that tells us that it was manual refund.